ui/.env

```diff
@@ -28,7 +28,7 @@ REACT_APP_MAX_ALLOWED_REPLICA=10
 REACT_APP_SENTRY_DSN=
 
 # Default Experiment Engine Remote UI settings.
-REACT_APP_DEFAULT_EXPERIMENT_ENGINE={"name":"","url":""}
+REACT_APP_DEFAULT_EXPERIMENT_ENGINE={"name":"","url":"","config":""}
```

Comment:
It would be good to document these properties in `config.js` - we take a single JSON string for this but expect some properties to be there which is not mentioned clearly anywhere in the code at the moment.

Eventually we also need to add info about the microfrontend configuration and the "default engine" here: https://github.com/gojek/turing/blob/main/engines/experiment/docs/developer_guide.md